src/main/scala/uk/gov/nationalarchives/tdr/api/utils/NaturalSorting.scala

```diff
@@ -0,0 +1,31 @@
+package uk.gov.nationalarchives.tdr.api.utils
+
+import scala.util.matching.Regex
+import java.text.Normalizer
+
+object NaturalSorting {
+  implicit object ArrayOrdering extends Ordering[Array[String]] { // 4
```

Comment:
I think you found this [on this site](https://rosettacode.org/wiki/Natural_sorting#Scala) Could you put a comment in with a link back to it. It's quite a difficult thing to follow and I think it would help to have the context from the site.

Reply:
Yeah, that make sense. Add in the doc comment.